bitcoin/src/blockdata/transaction.rs

```diff
@@ -263,6 +264,13 @@ impl TxIn {
         size + Sequence::SIZE
     }
 
+    /// Returns the base weight of this input.
+    ///
+    /// Base weight excludes the witness and script.
+    pub fn base_weight() -> Weight {
```

Comment:
Should be private const.

Reply:
Alright.  Originally I had implemented this as a const [here](https://github.com/rust-bitcoin/rust-bitcoin/pull/2053/commits/1b6f61e086b964be96c442959100bcc2e130d3d8).  Since then, I noticed [base_size](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/bitcoin/src/blockdata/transaction.rs#L257) has been added as a method, so I was following that design.

Reply:
added as a const https://github.com/rust-bitcoin/rust-bitcoin/pull/2230/commits/f7ef53a95241ccf3e887ceb40f1356ef2d063fce instead again.